packages/core-utils/src/httpie.ts

```diff
@@ -91,7 +94,7 @@ class Httpie {
             const { body, headers, statusCode } = await got[method](url, opts);
 
             return {
-                body: parseJSON(body).value,
+                body: JSON.parse(body || {}),
```

Comment:
```suggestion
                body: JSON.parse(body || "{}"),
```

`JSON.parse` expects a string, not an object, otherwise you'll get a `SyntaxError` as it will try to parse `[object Object]` from `{}.toString()`.